src/modules/jobs/models/tech-load.model.ts

```diff
@@ -0,0 +1,398 @@
+export class TechLoad {
+
+  public  jobs: JobData[];
+  private jobCount: number;
+  private uuid: string;
+
+  private seed: number;
+  private seedIndex: number = 0; // doesn't really matter what it's instantiated to, just needs to be any number >= 0 so the seed methods can generate the next number
+
+  constructor(uuid: string, amount?: number) { // Can be built with a given amount, but if not will seed the number of jobs to build
+    this.uuid = uuid;
+    this.seed = Math.abs(this.convertStrToNum(uuid));
```

Comment:
Add commenting explain the reason why Math.abs() being used

Reply:
@looksliketyler Wouldn't it be just making sure the number is positive?

Reply:
commented. it has to do with the fact that the seed increments++, so starting at a negative number can cause you to hit 0 which will always create the same number (because no matter what you multiply 0 by it'll always be 0)